docs/api-docs/apps/guide/apps-05-oauth.mdx

```diff
@@ -120,9 +120,12 @@ BigCommerce responds to the access_token request with JSON that contains a perma
   "scope": "store_v2_orders store_channel_listings_read_only",
   "user": {
     "id": 24654,
-    "username": "merchant@example.com",
     "email": "merchant@example.com"
   },
+  "owner": {
```

Comment:
Username should be in both user and owner. I provided an incomplete  object.
"username": "merchant@example.com",